zipkin-lens/.env.test

```diff
@@ -0,0 +1,2 @@
+BASE_PATH=/zipkin
+API_BASE=http://localhost:51204/zipkin
```

Comment:
is this .env.test file used anywhere? if not, maybe we can delete it to remove questions about it or possible port conflicts. if this actually used in tests, I guess there's no way to have it use a random port e.g. by specifying port zero?

Reply:
It is used during the tests so the test Browserrouter knows where to find the routes etc. I don't really know what happens if I skip the API_BASE variable. If you ever start with E2E tests you can then simply start Zipkin on port 51204.

Reply:
good enough for me!